client/src/components/GraphOverlay/GraphOverlay.js

```diff
@@ -2,32 +2,32 @@ import { Fragment } from "react";
 import { connect } from "react-redux";
 import MediaQuery from "react-responsive";
 
-import "./RotateLandscape.scss";
+import "./GraphOverlay.scss";
 import { IconRotatePhone, IconExpandWindowWidth } from "../../icons/icons.js";
 
-import { rotate_landscape_off } from "../../InfoBase/AppState.js";
+import { hide_graph_overlay } from "../../InfoBase/AppState.js";
 import { create_text_maker_component } from "../misc_util_components";
 
-import text from "./RotateLandscape.yaml";
+import text from "./GraphOverlay.yaml";
 
 const { TM, text_maker } = create_text_maker_component(text);
 
 class LandscapeOverlay extends React.Component {
   render() {
-    const { children, rotate_landscape, rotate_landscape_off } = this.props;
+    const { children, show_graph_overlay, hide_graph_overlay } = this.props;
```

Comment:
`show_graph_overlay, hide_graph_overlay` is still a bit weird to me lol sorry I think leaving `hide_graph_overlay` as-is and `is_showing_graph_overlay` would be good!